packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -392,4 +383,11 @@ export class LambdasNestedStack extends NestedStack {
 
     return outboundDocumentRetrievalLambda;
   }
+
+  private normalizePollingDuration(duration: Duration): string {
+    return Math.min(
+      duration.toMilliseconds(),
+      MAXIMUM_LAMBDA_TIMEOUT.minus(Duration.seconds(30)).toMilliseconds()
```

Comment:
Before it was giving 1min of buffer to the lambda, I think 30s is enough (and the impact of that on 11min timeout is lower than on 5min 😄 )

Reply:
And it's only applicable now if the timeout is equal to or greater than 15min (max AWS lambda duration).